Express-setup

yanzaxdev opened the pull request.

yanzaxdev committed:
Express setup + drizzle

yanzaxdev committed:
Express setup + drizzle

yanzaxdev committed:
Merge branch 'main' of https://github.com/yanzaxdev/forum-project into express-setup

yanzaxdev committed:
readme.md and .gitignore tweak

lioravigdor commented on backend/.vscode/settings.json:
```diff
@@ -0,0 +1,22 @@
+{
```
delete this file from repo and add .vscode/ to .gitignore

yanzaxdev committed:
deleting .vscode

yanzaxdev committed:
updating .gitignore

yanzaxdev committed:
In this commit:
- Added courseController.ts
- Added courseRoutes.ts
- Moves db schema folder to db/schema
- changed getLangFromServer to getLang
- Added tryCatch.ts to handle async errors
************************************
-Made first api call to get course data from express~

yanzaxdev committed:
tweak

yanzaxdev committed:
Tweak

yanzaxdev committed:
Merge pull request #3 from yanzaxdev/first-api-routes

First api routes
